test/storeTest.js

```diff
@@ -2167,6 +2167,36 @@ types.forEach(function (type) {
 
                   });
 
+                  if (type === 'redis') {
+                    describe('when events do not start at revision 0', function () {
+
+                      it('should return the events specified by the revMin and revMax', function (done) {
+
+                        var firstEventMatcher = options.prefix + ':' + options.eventsCollectionName + ':*:*:*:*:' + 'idWithAgg' + ':*:0'
+                        var revMin = 1
+                        var revMax = 2
+
+                        store.client.keys(firstEventMatcher, (err, keys) => {
```

Comment:
In order to supported legacy node versions, and avoid the use of transpilers, arrow functions should be avoided.